app/views/playlists/index.html.erb

```diff
@@ -35,7 +35,7 @@ Unless required by applicable law or agreed to in writing, software distributed
                 <th>Visibility</th>
                 <th>Created</th>
                 <th>Updated</th>
-                <th class="text-right">Actions</th>
+                <th class="text-right" data-orderable=false>Actions</th>
```

Comment:
Might want to keep attributes within double quotes `data-orderable="false"`, unless there's a specific reason not to: